python/cuml/dask/neighbors/kneighbors_classifier.py

```diff
@@ -63,18 +59,14 @@ class KNeighborsClassifier(NearestNeighbors):
     verbose : int or boolean, default=False
         Sets logging level. It must be one of `cuml.common.logger.level_*`.
         See :ref:`verbosity-levels` for more info.
-    output_type : {'input', 'cudf', 'cupy', 'numpy', 'numba'}, default=None
-        Variable to control output type of the results and attributes of
-        the estimator. If None, it'll inherit the output type set at the
-        module level, `cuml.global_output_type`.
-        See :ref:`output-data-type-configuration` for more info.
     """
     def __init__(self, client=None, streams_per_handle=0,
-                 verbose=False, **kwargs):
+                 verbose=False, batch_size=1024, **kwargs):
```

Comment:
Do we need to add this to the init explicitly? Could we just document it but leave it implicit in **kwargs?

Reply:
IMO, that would be ideal- just avoid hading to duplicate it altogether.